erpnext/manufacturing/doctype/production_planning_tool/production_planning_tool.py

```diff
@@ -200,15 +200,19 @@ def get_distinct_items_and_boms(self):
 
 	def create_production_order(self, items):
 		"""Create production order. Called from Production Planning Tool"""
-		from erpnext.manufacturing.doctype.production_order.production_order import OverProductionError
-
+		from erpnext.manufacturing.doctype.production_order.production_order import OverProductionError, get_default_warehouse
+		warehouse = get_default_warehouse()
 		pro_list = []
 		for key in items:
 			pro = frappe.new_doc("Production Order")
 			pro.update(items[key])
 			pro.set_production_order_operations()
-
+			if warehouse:
+				pro.wip_warehouse = warehouse.get('wip_warehouse')
```

Comment:
WIP Warehouse is not taken from Production Planning Tool. Only FG warehouse gets pulled if specified in Sales Order